students/roviram/Lesson02/FibonacciSeries.py

```diff
@@ -0,0 +1,80 @@
+#-----------------------------------------------------------#
+# Dev: Miguel Rovira-Gonzalez
+# Desc: Fibnonacci Series Homework, Week 2
+# Date Created: Sunday 4/21
+#-----------------------------------------------------------#
+def fibonacci(n):
+    """"
+    Compute the nth Fibnonacci number
+    fib(n) = fib(n-2) + fib(n-1)
+    """
+
+    a = 0
+    b = 1
+    for number in range(n):
+        a, b = b, a + b
+    return a
```

Comment:
did you notice that you are computing an extra one, then returning the second to last?

but you do get the right answer!